### PR TITLE
Don't Require PrimInt63 and PrimFloat by default

### PR DESCRIPTION
**Pull request by proux01**
Some users are complaining that loading elpi loads part of the stdlib with many axioms even if they are not using them.
This PR tries to improve on that point.

### CODE REVIEW

File: src/coq_elpi_vernacular_syntax.mlg [hunk 1]
```diff
@@ -84,11 +93,11 @@ GRAMMAR EXTEND Gram
 
   term: LEVEL "0"
     [ [ "lib"; ":"; id = qualified_name -> {
-          let ref = Coqlib.lib_ref (String.concat "." (snd id)) in
+          let ref = lib_ref id in
           let path = Nametab.path_of_global ref in
```
Comment (SkySkimmer) [inline src/coq_elpi_vernacular_syntax.mlg:97]: not really related to this PR but looking up lib_ref and nametab during parsing is pretty bad with parsing/execution separated

Comment (gares) [reply src/coq_elpi_vernacular_syntax.mlg:97]: Indeed...

Comment (gares) [reply src/coq_elpi_vernacular_syntax.mlg:97]: Actually, this is really tricky. I do extend the Coq grammar, so there is a bug, but we only use it inside elpi code, hence inside a quotation. The text between `{{ ... }}` is parsed as a string, and the Coq parser is called at run time. So I think it is mostly fine for the use we make.

Anyway, `lib:` predates the ability of elpi to compile units, we don't need that anymore. So maybe it is just better to wipe it out.

Comment (proux01) [reply src/coq_elpi_vernacular_syntax.mlg:97]: Maybe should we deprecate it first? In any case, that should probably happen in another PR.

**Comment by proux01** [comment]
@gares CI green

**Pull request merged**